### PR TITLE
add !choice command

### PR DESCRIPTION
**Pull request by viraxor**
This command returns a random choice from a given list of choices separated by a semicolon. #98 

Example:
```
<viraxor> !choice wildchip; allgear; s3xmodit
<BotB2_tes> allgear
<viraxor> !choice ohb; 2hb; 4hb
<BotB2_tes> ohb
```

### CODE REVIEW

**Comment by langel** [comment]
please open a new issue in the issue tracker and label it as an enhancement before submitting pull requests so other folks who are involved in the code base can discuss the merit and design of the feature

File: bot_modules/irc_commands.js [hunk 1]
```diff
@@ -107,6 +107,19 @@ module.exports = {
...
+	
+	/**
+	 * 	choice
+	 *
+	 */
+	choice: (info, words) => {
+		let choice_words = words.slice(1).join(' ')
+	    
+		if (choice_words.includes(";")) {
+			let choices = choice_words.split(";")
```
Comment (langel) [inline bot_modules/irc_commands.js]: why a semicolon? comma makes more sense

File: bot_modules/irc_commands.js [hunk 2]
```diff
@@ -107,6 +107,19 @@ module.exports = {
...
+			bot.say(info.channel, choices[Math.floor(Math.random() * choices.length)].trim())
+		} else {bot.say(info.channel, "there's nothing to choose from! =X")}
```
Comment (langel) [inline bot_modules/irc_commands.js]: please put else on its own line after the if closing bracket
either put the one-liner following the else on its own line or don't add the curly brackets

**Comment by langel** [comment]
also, please use semicolons at the end of lines
when this project started all lines had semicolons
someone removed them all and merged their code w/o consulting me many years ago 
so now i am the only one that can merge code

File: bot_modules/irc_commands.js (hunk 1 above)
Comment (viraxor) [reply bot_modules/irc_commands.js]: I was thinking that people might use commas in sentences

**Pull request closed without merging**